libs/filabridge/include/filament/MaterialEnums.h

```diff
@@ -115,9 +115,12 @@ enum VertexAttribute : uint8_t {
     UV1             = 4, //!< texture coordinates (float2)
     BONE_INDICES    = 5, //!< indices of 4 bones, as unsigned integers (uvec4)
     BONE_WEIGHTS    = 6, //!< weights of the 4 bones (normalized float4)
+    CUSTOM0         = 7, //!< the first custom attribute (this is always last in the enum)
```

Comment:
Is this how clients would set attributes >= 1?

```
VertexBuffer::Builder()
.attribute(VertexAttribute(VertexAttribute::CUSTOM0 + 1) , 0, VertexBuffer::AttributeType::UBYTE4, 8, 12)
```

Reply:
Yes, exactly.  Hmm maybe I should simply enumerate 8 custom values...

Reply:
I added another commit for this, if you guys don't like the explicit enumeration then I can remove it from the PR.